### PR TITLE
Fixed Windows executable path logic and refactoring.

### PR DESCRIPTION
**Pull request by matracey**
- Fixed Windows executable path logic based on path changes since Edge Chromium GA.
    - For reference, Edge, Beta, and Dev are installed to Program Files; Canary is installed to AppData.
 - Updated readme and changelog.
    - Reset changelog to capture Edge specific changes.
 - Added eslint for enforcing modern JavaScript standards and practices.
 - Fixed ESLint errors.
 - Reordered common functions to top of index file.
 - Removed Chromium launcher.
 - Removed unused getBin function for linux.
    -  Edge is not available on Linux at this time.
 - Refactored Edge browser launcher.
 - Refactored Edge Canary browser launcher.
 - Added Edge Beta launcher.
 - Added Edge Dev launcher.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,204 +1,8 @@
-<a name="3.1.0"></a>
```
Comment (koddsson) [inline CHANGELOG.md]: Can we keep the changelog? It provides historical info after we forked from the chrome launcher.

File: README.md [hunk 2]
```diff
@@ -54,14 +54,27 @@ You can pass list of browsers as a CLI argument too:
...
+
 #### Usage
 ```bash
 $ npm i -D puppeteer karma-edgium-launcher
 ```
 
 ```js
 // karma.conf.js
-process.env.CHROME_BIN = require('puppeteer').executablePath()
+process.env.EDGE_BIN = require('puppeteer').executablePath()
```
Comment (koddsson) [inline README.md:77]: Nice!

File: package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "karma-edgium-launcher",
-  "version": "3.1.0",
-  "description": "A Karma plugin. Launcher for Edge and Edge Canary.",
+  "version": "3.1.1",
+  "description": "A Karma plugin. Launcher for Microsoft Edge and Edge Canary.",
   "main": "index.js",
   "scripts": {
-    "lint": "standard",
+    "lint": "eslint .",
```
Comment (koddsson) [inline package.json]: I'd prefer to keep the linting system we already had instead of changing it now.

File: CHANGELOG.md (hunk 1 above)
Comment (matracey) [reply CHANGELOG.md]: No problem. I've restored the changelog now.

File: package.json (hunk 3 above)
Comment (matracey) [reply package.json]: Fair point! I've restored standard as the linter.

**Comment by matracey** [comment]
I've updated my fork based on the requested changes and the build is passing now. All looks good to merge and release to npm.

**Comment by PVermeer** [comment]
@koddsson Can this be merged? Seems to work fine.

**Comment by Eikir** [comment]
With this fix Edge can finally be started again. Can this be merged, please?

**Pull request closed without merging**